data-migration/2.x-to-3.x/README.md

```diff
@@ -0,0 +1,81 @@
+# Updating data from 2.x to 3.x
+
+The following code snippet emulates the situation where the code is in version 3.0.0 while the data is in version 2.0.2
+(this assumes that you've followed the [instructions for removing local
+state](https://github.com/research-software-directory/research-software-directory/blob/3.0.0/docs/dev.md#removing-local-state)):
+
+```shell
+# give me the code state for 3.0.0 ...
+git checkout 3.0.0
+
+# ... but with sample data from version 2.0.2
+git checkout 2.0.2 -- database/db-init
```

Comment:
You should only do this when you don't have a 2.x database
```suggestion
# ... if you don't have the database filled with 2.x version data
# it can be filled with sample data from version 2.0.0 using 
git checkout 2.0.2 -- database/db-init
```

Reply:
Also this checkout leaves your local clone with staged changes, you should be careful not to commit those changes.  To undo I had to run

```shell
git restore --staged database/db-init
git restore database/db-init
```

A note would be nice.

Reply:
I updated notes to include:
```
git reset HEAD database/db-init
git checkout -- database/db-init
```
which have the effect of rolling back the changes to db-init resulting from the checkout

Reply:
git v2.23.0 introduced `git restore` to replace `git checkout` in some places. So I used the commands my git suggested. Using these older commands is fine, as they work in more versions of git.

Reply:
Aha I didnt know that, and indeed I'm using git v2.21.0